.github/workflows/release-patch.yml

```diff
@@ -0,0 +1,91 @@
+name: Release 1b - Auto Patch
+run-name: Patch Release ${{ inputs.version }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Major Metabase version (e.g. 45, 52, 68)'
+        type: number
+        required: true
+  schedule:
+    - cron: '0 7 * * 1-5' # every weekday at 8am/9am Central European Time
```

Comment:
@perivamsi or @paoliniluis any preferences for when the nightly build should happen?

Reply:
8 pm ET